brave/browser-laptop
# Adds referral code read into browser

Submitter Checklist:


Resolves #12789
Resolves #12790

Test Plan:
1) clean profile
2) create empty folder `brave-development` and create file `promoCode` with `123456` inside
3) start browser `LEDGER_ENVIRONMENT=staging npm run start`

Reviewer Checklist:


Tests
